frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -265,23 +366,41 @@ function Application(): JSX.Element {
 					</Button>
 
 					<Card>
-						<GraphTitle>Error Percentage</GraphTitle>
-						<GraphContainer>
-							<FullView
-								name="error_percentage_%"
-								fullViewOptions={false}
-								onClickHandler={handleGraphClick('Error')}
-								widget={errorPercentageWidget}
-								yAxisUnit="%"
-								onDragSelect={onDragSelect}
-							/>
-						</GraphContainer>
+						{topLevelOperationsIsError && (
+							<Typography>
+								{axios.isAxiosError(topLevelOperationsError) &&
```

Comment:
what if this is not axios error ?

Reply:
Will set the Something went wrong message.